sigs/sig-marketing/README.md

```diff
@@ -0,0 +1,30 @@
+# Argo Sig Marketing
+
+Argo Sig Marketing coordinates efforts between interested parties to promote, advocate, and grow the Argo Project and community. This includes social media, subreddits, events, branding, and all other marketing activities.
+
+## Meetings
+Sig marketing meetings are held twice monthly. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com&ctz=America%2FDenver).
+
+## Membership Requirements 
+- Members are nominated and approved by Argo Proj maintainers. 
+- Must represent the open-source community’s best interest with a vendor-neutral practice.
+- Must follow the CNCF rules and guidances closely.
+- Must have shown a proven record of enabling the open-source community by marketing.
+- Technical marketing is preferred.
+- Cannot have more than two members from the same company.
```

Comment:
I dont think we need to limit it to two / company. Matters should be consensus driven and for anything that cant reach consensus, it should be brought to and follow maintainer voting procedures. 
Membership here is separate than being a maintainer, so adding to SIG Marketing wont add to the maintainer group.